python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -641,9 +674,21 @@ class BuiltinMethodValue extends FunctionValue {
         )
     }
 
-    override int minParameters() { none() }
+    override string descriptiveString() { result = "builtin-method " + this.getQualifiedName() }
+
+    override int minParameters() {
+        none()
+    }
```

Comment:
how come `minParameters` and `maxParameters` have been expanded to 3 lines? If it's caused by autoformatting, then we need to figure out what is going on, since _my_ autoformatter (which I rebuild a few days ago) is rewriting these 3 lines to become 1 again.

How are you invoking the autoformatter? If it's through VS Code, are you using your own codeql installation? (or the one that VS Code will automatically download). can you try to update the autoformatter and run it again?

Reply:
Currently I'm using `qlformat` but it might not be up to date. I'll update and re-autoformat.

Reply:
Done!